charts/caraml/values.yaml

```diff
@@ -20,7 +20,7 @@ global:
     serviceName: caraml-authz
   mlp:
     postgresqlDatabase: mlp
-    apiPrefix: "/v1"  # path prefix to where mlp endpoints lie if running on localhost
```

Comment:
@pradithya.pura: 

continued: https://github.com/caraml-dev/helm-charts/pull/258#discussion_r1205395284 

This did cause a problem. there was a `//` related error when I was testing in tkpd staging. I saw this when Turing tried to make a call with mlp. 

We didn't find this earlier because this would come up when using the umbrella chart that has the globals defined. Using individual charts wouldn't cause this error since the default chart values doesn't have any globals defined related to this. 

This apiPrefix needs to be set to an empty string for the right usage.

Reply:
Noted @maniteja6799 ! How was it not a problem for Merlin?